libraries/AP_Scripting/applets/mount-viewpro-driver.lua

```diff
@@ -0,0 +1,372 @@
+-- mount-viewpro-driver.lua: Viewpro mount/gimbal driver
+--
+-- How to use
+--   Connect gimbal UART to one of the autopilot's serial ports
+--   Set SERIALx_PROTOCOL = 28 (Scripting) where "x" corresponds to the serial port connected to the gimbal
+--   Set SCR_ENABLE = 1 to enable scripting and reboot the autopilot
+--   Set MNT1_TYPE = 9 (Scripting) to enable the mount/gimbal scripting driver
+--   Install this script on the autopilot's SD card in the APM/scripts directory and reboot the autopilot
```

Comment:
```suggestion
--   Copy this script to the autopilot's SD card in the APM/scripts directory and reboot the autopilot
```
Isn't install just a copy operation?